keras_core/saving/saving_api_test.py

```diff
@@ -0,0 +1,190 @@
+import os
+import unittest.mock as mock
+
+import numpy as np
+from absl import logging
+
+from keras_core import layers
+from keras_core.models import Sequential
+from keras_core.saving import saving_api
+from keras_core.testing import test_case
+
+
+class SaveModelTests(test_case.TestCase):
+    def setUp(self):
+        self.model = Sequential(
+            [
+                layers.Dense(5, input_shape=(3,)),
+                layers.Softmax(),
+            ],
+        )
+        self.filepath = "test_model.keras"
```

Comment:
You can use a temporary directory for this, e.g. `self.get_temp_dir`. Then no need to tear down. Also you don't really need the `setUp` method (it's complex), you can just have a `get_model()` utility

Reply:
Done